torch/distributed/__init__.py

```diff
@@ -16,3 +18,4 @@ def is_available():
     # this.
     from .distributed_c10d import _backend  # noqa: F401
     from .rpc import *  # noqa: F401
+    from .internal_rpc_utils import *  # noqa: F401
```

Comment:
If rpc is Python 3 only, you really shouldn't import this (or `.rpc`) if you're on Python 2, right?

Also, why are we importing all of `internal_rpc_utils` into the PUBLIC module `torch.distributed`? That makes no sense to me. If they are internal, please access them through full module qualification; don't pollute public namespace just for the sake fo convenience.